src/cleaners/date-published.test.js

```diff
@@ -1,13 +1,13 @@
 import assert from 'assert';
-import moment from 'moment-timezone';
+import dayjs from 'dayjs';
 
 import cleanDatePublished, { cleanDateString } from './date-published';
 
 describe('cleanDatePublished(dateString)', () => {
   it('returns a date', () => {
     const datePublished = cleanDatePublished('published: 1/1/2020');
 
-    assert.equal(datePublished, moment('1/1/2020', 'MM/DD/YYYY').toISOString());
+    assert.equal(datePublished, dayjs('1/1/2020', 'M/D/YYYY').toISOString());
```

Comment:
I know this is how the old tests were too, but can we change all these expected dates into hard-coded strings? I think that would make the functionality we're testing more clear: we expect strings of varying format and quality, and we always return a string in a specific format.

So this line would become:

```js
assert.equal(datePublished, '2020-01-01T00:00:00.000Z');
```

I think with that change we could probably remove dayjs as a dependency from this test file entirely.

Reply:
exactly, i'll have a commit with these changes shortly ✅